composer.json

```diff
@@ -28,7 +28,7 @@
         }
     },
     "require-dev": {
-        "sabre/cs": "~1.0.0",
+        "friendsofphp/php-cs-fixer": "^2",
         "phpunit/phpunit" : "*"
     },
     "config" : {
```

Comment:
Tbh i dont like the non standard bin-dir.. but I guess we keep it for consistency